sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/MessageUtils.java

```diff
@@ -326,4 +330,38 @@ public static ServiceBusMessage traceMessageSpan(ServiceBusMessage serviceBusMes
         }
         return serviceBusMessage;
     }
+
+    /**
+     * Convert described type to URI object.
+     * @param described the described of described type.
+     * @return URI or if create failed return String.
+     */
+    public static Object describedToURI(Object described) {
+        try {
+            return URI.create((String) described);
```

Comment:
I am thinking about why not to move these `describedTo*()` methods to each concrete classes?
if it is posible to use like `((UriDescribedType)describedType).getOriginObject()` to return a URI.

Reply:
Can not cast to concrete class, we received a private class named `UnknownDescribedType` implements `DescribedType` in `DecoderImpl`.